polytiming.js

```diff
@@ -1,17 +1,28 @@
 (function() {
   'use strict';
 
-  let measuredElements = new Set();
-  let measuredMethods = new Set();
-  let configuredMethods = window.location.search.slice(1).split('&')
-      .map(function(part) {
-        return part.split('=');
-      }).reduce(function(l, r) {
-        if (r[0] === 'instrumentPolymer') {
-          return l.concat(r[1].split(','));
-        }
-        return l;
-      }, []);
+  function setOfQueryParams(paramName) {
+    return new Set(window.location.search.slice(1).split('&')
+        .map(function(part) {
+          return part.split('=');
+        }).reduce(function(l, r) {
+          if (r[0] === paramName) {
+            return l.concat(r[1].split(','));
+          }
+          return l;
+        }, []));
+  }
+
+  const measuredElements = new Set();
+  const measuredMethods = new Set();
+  const configuredMethods = setOfQueryParams('instrumentPolymer')
+  const trackedElements = setOfQueryParams('trackElement')
+  let shouldTrackElement = (elementName) => true;
```

Comment:
Given the performance sensitive nature of where this check is used, I wonder if it would be more practical to have a plain object mapping of tag name to boolean.